### DISCUSSION TITLE
Moving the data from staging to production [Jun 28]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2571] [Discord Threads]
> 🙋‍♂️ **zblesk**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### if all your tables were on NC_DB - it will be a simple pg_dump.. you could give that a try  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/991058103418097664)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
